### ISSUE TITLE
Specify notations in signatures

### ISSUE THREAD

**Issue by cyrus-** [cyrus-/relit#11]

**Comment by charlesetc** [comment]
To be clear you *can* specify notation aliases in signatures, which is something but a weird state to only support.

**Comment by cyrus-** [comment]
yeah, you should be able to write down the .rei file corresponding to Regex_notation.re for example
